README.md

```diff
@@ -209,7 +209,7 @@ Step 1-3 are to be done on the host OS (Linux):
     ```bash
     # Two methods:
     # (1) Create softlinks on host
-    mkdir -p /dev/sgx
+    mkdir -p /dev/sgx && cd /dev/sgx
     ln -sf ../sgx_enclave /dev/sgx/enclave
```

Comment:
Is it better to change the next lines to below lines?
ln -sf /dev/sgx_enclave /dev/sgx/enclave
ln -sf /dev/sgx_provision /dev/sgx/provision

Reply:
The reason I stick to the reletive links was I found in /dev/char/ and /dev/block, all were relative links as well.